lib/roo_on_rails/railties/new_relic.rb

```diff
@@ -12,6 +12,15 @@ class NewRelic < Rails::Railtie
 
           abort 'Aborting: NEW_RELIC_LICENSE_KEY is required' if license_key.nil?
 
+          # Report application stats to a per-service (worker, web) New Relic app, and to a main
+          # application for all services.
+          base_name = ENV['BASE_NEW_RELIC_APP_NAME']
+          service_name = ENV['HOPPER_SERVICE_NAME']
+          if !base_name.nil? && ENV['NEW_RELIC_APP_NAME'].blank?
+            task_app_name = !service_name.nil? ? "#{base_name} - #{service_name}" : nil
```

Comment:
Might be more readable with `service_name.present?`

Reply:
Ahh, yes! :-) Much nicer.